src/ocrd_network/cli/client.py

```diff
@@ -179,6 +180,7 @@ def check_workflow_job_status(address: Optional[str], workflow_job_id: str):
                                 'the "OCRD_NETWORK_SERVER_ADDR_PROCESSING" env variable is used by default')
 @click.option('-m', '--path-to-mets', required=True)
 @click.option('-w', '--path-to-workflow', required=True)
+@click.option('-p/-P', '--page-wise/--no-page-wise', is_flag=True, default=False, help="Whether to generate per-page jobs")
 @click.option('-b', '--block', default=False, is_flag=True,
               help='If set, the client will block till job timeout, fail or success.')
 @click.option('-p', '--print-state', default=False, is_flag=True,
```

Comment:
I have not paid attention to the shortcut `-p/-P` and introduced it again for `--print-state`. Just replace that, the rest is good.

Reply:
Oh right, yeah, we probably don't even need the short versions. I think selecting `--page-wise` vs `--no-page-wise` should be a very conscious decision worth typing a long option.

Reply:
Resolved with [854403d](https://github.com/OCR-D/core/pull/1278/commits/854403de6ea880c31b82463bba3850c07565327d)